its/projects/TwoDrivesThreeProjects/DefaultDrive/Program.cs

```diff
@@ -0,0 +1,3 @@
+﻿// See https://aka.ms/new-console-template for more information
+string x = null;
+Console.WriteLine($"Hello, World! {x.ToString()}");
```

Comment:
Just to reduce scaffolding noise. 
```suggestion
Console.WriteLine("Hello, World!");
```
If we need to assert an issue, this is a good trick (clear, easy to understand):
```
// FIXME: This should raise S1134 
```